lib/createJestRunner.ts

```diff
@@ -9,7 +9,7 @@ import type {
   TestRunnerOptions,
   TestWatcher,
 } from 'jest-runner';
-import { Worker, JestWorkerFarm } from 'jest-worker';
+import { Worker, type JestWorkerFarm } from 'jest-worker';
```

Comment:
The `type` modifiers on import names feature was added in TypeScript 4.5. Should be fine. Or?

Reply:
yeah - unless the import is included in the built `.d.ts`, it doesn't matter what we use